packages/groff/Makefile.in.patch

```diff
@@ -1,195 +1,12 @@
- 	  test -z "$$dir" || $(MKDIR_P) "$$dir"; \
- 	done
- install: $(BUILT_SOURCES)
-@@ -11528,7 +11459,6 @@
- # vim: set filetype=automake textwidth=72:
- $(hdtblexample_DATA): $(HDTBLSTRIPFILES)
- @BUILD_EXAMPLES_TRUE@contrib/hdtbl/examples/test-hdtbl.sh: $(top_builddir)/config.status \
--@BUILD_EXAMPLES_TRUE@	$(HDTBLPROCESSEDEXAMPLEFILES) $(top_srcdir)/contrib/hdtbl/examples/test-hdtbl.sh.in
- @BUILD_EXAMPLES_TRUE@	sed -e "s|[@]abs_top_builddir[@]|$(abs_top_builddir)|g" \
- @BUILD_EXAMPLES_TRUE@	    -e "s|[@]GHOSTSCRIPT[@]|$(GHOSTSCRIPT)|g" \
- @BUILD_EXAMPLES_TRUE@	    	$(top_srcdir)/contrib/hdtbl/examples/test-hdtbl.sh.in > $@
-@@ -11545,9 +11475,6 @@
- 	&& sed -e "s|[@]fontdir[@]|$(fontdir)|" \
- 	       -e "s|[@]EGREP[@]|$(EGREP)|" $< >$@
- 
--$(HDTBLPROCESSEDEXAMPLEFILES): gnu.eps groff troff pic tbl \
--  contrib/hdtbl/examples/common.roff
--
- $(HDTBLSTRIPFILES): $(top_srcdir)/tmac/strip.sed
- 	@$(MKDIR_P) `dirname $@`
- 	sed -f $(top_srcdir)/tmac/strip.sed $(hdtbl_srcdir)/`basename $@`-u > $@;
-@@ -11838,20 +11765,6 @@
- @BUILD_HTMLEXAMPLES_TRUE@	  $(DESTDIR)$(exampleimagedir)
- 
- uninstall-hook: uninstall_doc_examples uninstall_doc_htmldoc uninstall_mom
--uninstall_doc_examples:
--@BUILD_EXAMPLES_TRUE@	$(RM) -f $(DESTDIR)$(exampledir)/gnu.eps
--@BUILD_HTMLEXAMPLES_TRUE@	-test -d $(DESTDIR)$(docexamplesdir) && \
--@BUILD_HTMLEXAMPLES_TRUE@	cd $(DESTDIR)$(docexamplesdir) && \
--@BUILD_HTMLEXAMPLES_TRUE@	for f in $(HTMLEXAMPLEFILESALL); do \
--@BUILD_HTMLEXAMPLES_TRUE@	  $(RM) $$f; \
--@BUILD_HTMLEXAMPLES_TRUE@	done
--@BUILD_HTMLEXAMPLES_TRUE@	-test -d $(DESTDIR)$(docexamplesdir)/$(imagedir) && \
--@BUILD_HTMLEXAMPLES_TRUE@	cd $(DESTDIR)$(docexamplesdir)/$(imagedir) && \
--@BUILD_HTMLEXAMPLES_TRUE@	for f in $(HTMLEXAMPLEIMAGEFILES); do \
--@BUILD_HTMLEXAMPLES_TRUE@	  $(RM) $$f; \
--@BUILD_HTMLEXAMPLES_TRUE@	done
--@BUILD_HTMLEXAMPLES_TRUE@	-rmdir $(DESTDIR)$(docexamplesdir)/$(imagedir)
--@BUILD_HTMLEXAMPLES_TRUE@	-rmdir $(DESTDIR)$(docexamplesdir)
- 
- uninstall_doc_htmldoc:
- @BUILD_HTML_TRUE@	-test -d $(DESTDIR)$(htmldocdir) && \
+--- groff-1.22.4/Makefile.in	2021-10-02 10:46:19.717453972 +0300
++++ groff-1.22.4.mod/Makefile.in	2021-10-02 16:50:23.231359302 +0300
+@@ -3350,7 +3350,7 @@
+ YACC = @YACC@
+ abs_builddir = @abs_builddir@
+ abs_srcdir = @abs_srcdir@
+-abs_top_builddir = @abs_top_builddir@
++abs_top_builddir = /home/builder/.termux-build/groff/host-build
```

Comment:
No hardcoded paths referencing build environment. You don't know where the termux-packages build could run, there no guarantee that this would be Docker image. `$TERMUX_TOPDIR` variable is user-configurable.

Update `$PATH` in one of build steps instead.

Reply:
Ok